docs/README.md

```diff
@@ -83,7 +83,7 @@ Open the `src/App.vue` component. Then replace the whole beginning of the file,
     <ais-search-box />
     <ais-results>
       <template scope="{ result }">
-        <h2><ais-highlight :result="result" attribute-name="name" /></h2>
+        <h2 v-html="result._snippetResult.name.value"></h2>
```

Comment:
For now this is an engine limitation.
Sanitizing this is a pain, and adding a comment will probably do more harm than be useful.
I think we can ship a "results sanitizer" as a decoupled piece that would just do the heavy lifting, but this needs some work & time that I don't currently have :/

WDYT?